src/components/KubeCtlShellAccess.js

```diff
@@ -16,6 +16,7 @@ const useStyles = makeStyles((theme) => ({
     marginRight: "5px",
     marginTop: "4px",
     lineHeight: "10px",
+    height: "25px"
```

Comment:
This is looking weird on cluster view page:
![Screenshot from 2022-10-07 12-03-48](https://user-images.githubusercontent.com/16691314/194482898-12fdb718-9252-49dd-9ad0-98078ac21e5f.png)

Reply:
reduced the height to 20 so that it looks consistent.